web/static/css/app.css

```diff
@@ -165,14 +171,34 @@ a {
 }
 
 .bottom__offset {
-  background-color: #47B9B4;
+  background-color: white;
   padding: 20px;
 }
 
 .bottom__title {
   font-size: 35px;
 }
 
+.bottom__back-link {
```

Comment:
yeah, this is how @eschaefer set things up for now, as it's just one or two pages